### PR TITLE
fix certificate contents

### PR DESCRIPTION
**Pull request by akihikokuroda**
### Summary
Fix: #910

### CODE REVIEW

File: charts/quantum-serverless/charts/gateway/templates/rayclustertemplate.yaml [hunk 1]
```diff
@@ -303,17 +306,6 @@ data:
                   cpu: {{ .Values.application.ray.cpu }}
                   memory: {{ .Values.application.ray.memory }}Gi
               securityContext:
-              volumeMounts:
```
Comment (IceKhan13) [inline charts/quantum-serverless/charts/gateway/templates/rayclustertemplate.yaml:306]: we do not need counts for worker anymore?

Comment (akihikokuroda) [reply charts/quantum-serverless/charts/gateway/templates/rayclustertemplate.yaml:306]: There were 2 volumeMounts. The latter was used.  This PR merges them into one.

File: charts/quantum-serverless/charts/gateway/templates/raycertgen.yaml [hunk 2]
```diff
@@ -232,4 +234,54 @@ data:
     kubectl get secret $CLUSTER_NAME -o=jsonpath='{.data.ca\.crt}' | base64 -d > $BASE_DIR/ca.crt
     kubectl get secret $CLUSTER_NAME -o=jsonpath='{.data.tls\.crt}' | base64 -d > $BASE_DIR/tls.crt
     kubectl get secret $CLUSTER_NAME -o=jsonpath='{.data.tls\.key}' | base64 -d > $BASE_DIR/tls.key
+  gencert_cert_worker.sh: |
+    #!/bin/sh
+    BASE_DIR=$1
+    CLUSTER_NAME=$2
+    IP_ADDRESS=$3
+    NAMESPACE=$4
```
Comment (Tansito) [inline charts/quantum-serverless/charts/gateway/templates/raycertgen.yaml:242]: I'm trying to understand what the pull request is doing Aki, so correct if I'm wrong. `NAMESPACE` argument is not needed in this case, isn't it? At least by the script.

**Review by IceKhan13 — APPROVED** [review]
Checked on rancher, everything works! Thank you!

**Pull request merged**